main.rb

```diff
@@ -0,0 +1,175 @@
+# rubocop:disable Metrics
+require './book'
+require './classroom'
+require './rental'
+require './person'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @people = []
+    @book = []
+    @rentals = []
+    @class = Classroom.new('class7')
+  end
+
+  def start
+    puts 'Welcome tp School Library App'
+    menu
+  end
+
+  def menu
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    user_option = gets.chomp
+    get_user_option(user_option)
+  end
+
+  def get_user_option(option)
+    case option
+    when '1'
+      all_books
+    when '2'
+      all_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      rentals_by_id
+    when '7'
+      'yes'
+    else
+      puts 'Please enter a number between 1 and 7'
```

Comment:
- [ ] An error is thrown when a correct number is selected after an incorrect number option is entered. Kindly ensure to fix this error.

**Error:**
![image](https://user-images.githubusercontent.com/12745474/151547416-82080365-f975-419c-a213-d9622211c03b.png)